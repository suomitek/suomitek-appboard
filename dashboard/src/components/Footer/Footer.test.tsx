import { shallow } from "enzyme";
import * as React from "react";

import Footer from "./Footer";

it("renders version and link to release", () => {
  const version = "v1.5.5-T500";
  const linkURI = `https://github.com/kubeapps/kubeapps/releases/tag/${version}`;

  const wrapper = shallow(<Footer appVersion={version} />);

  const link = wrapper.find("p.version-link a").first();
  expect(link.text()).toBe(version);
  expect(link.props().href).toBe(linkURI);
  expect(link).toMatchSnapshot();
});
