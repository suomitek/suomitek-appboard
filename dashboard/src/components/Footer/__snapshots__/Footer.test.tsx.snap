// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders version and link to release 1`] = `
<a
  className="type-color-white"
  href="https://github.com/kubeapps/kubeapps/releases/tag/v1.5.5-T500"
  rel="noopener noreferrer"
  target="_blank"
>
  v1.5.5-T500
</a>
`;
